promscale/guides/resource-recomm.md

```diff
@@ -35,7 +35,7 @@ remote_write:
   remote_timeout: 100s
```

Comment:
I'm not sure why this file exists, may be we should remove this? @VineethReddy02

Reply:
Looks like we didn't drop this file when we moved it from `guides` to `recommendations`. @arajkumar please drop the file if there is no reference for the same. 

Also, can add information on what are shared buffers and how users can allocate it?

Reply:
Sure, I'm adding it.